tree:
  ros__parameters:
    sim: False
    gripper_open_pos: 0.0
    gripper_close_pos: 0.8
    gripper_open_force: 100.0
    gripper_close_force: 100.0
    init_config: [0.0, -1.570796, 1.570796, -1.570796, -1.570796, 0.785]
    pose_srv_channel: "/get_object_pose"
    grasp_pose_srv_channel: "/get_object_grasp_pose"
    height_srv_channel: "/get_object_height"
    rnd_pose_srv_channel: "/get_object_rnd_pose"
    close_pose_srv_channel: "/get_object_close_pose"
    world_frame: "world"
    arm_base_frame: "base_link"
    grasp_offset_z: 0.02
    top_offset_z: 0.15
